go.mod

```diff
@@ -3,6 +3,8 @@ module github.com/lyft/flyteadmin
 go 1.13
 
 require (
+	cloud.google.com/go v0.56.0
```

Comment:
This is the last version staying on protobuf v1.3.5. Since v1.4.0, `github.com/golang/protobuf` started to depend on `google.golang.org/protobuf`: https://github.com/golang/protobuf/blob/v1.4.0/ptypes/timestamp/timestamp.pb.go#L9

So sooner or later we will need to face this issue again.

Reply:
oh this is awesome, thank you for finding it